skywalking-sniffer/skywalking-api/src/main/java/com/a/eye/skywalking/api/Tracing.java

```diff
@@ -21,4 +24,34 @@ public static String getTraceId() {
 
         return formatTraceId(spanData.getTraceId());
     }
+
+
+    /**
+     * Get the latest span of current trace.
+     *
+     * @return span. if it exists, or null
+     */
+    public static Span getLatestSpan() {
+        Span spanData = CurrentThreadSpanStack.peek();
+        return spanData;
+    }
+
+    /**
+     * set tag to the latest span of current trace
+     *
+     * @param tags tags map
+     */
+    public static void setTags2CurrentSpan(Map<String, String> tags) {
```

Comment:
The method should declare as
`public static void setTag(Span span, String tagKey, String tagValue)`